configs/detection/ava/README.md

```diff
@@ -80,6 +80,26 @@ python tools/train.py configs/detection/AVA/slowonly_kinetics_pretrained_r50_8x8
 
 For more details and optional arguments infos, you can refer to **Training setting** part in [getting_started](/docs/getting_started.md#training-setting) .
 
+### Train Custom Classes From Ava Dataset
+
+You can train custom classes from ava. Ava suffers from class imbalance. There are more then 100,000 samples for classes like `stand`/`listen to (a person)`/`talk to (e.g., self, a person, a group)`/`watch (a person)`, whereas half of all classes has less than 500 samples. In most cases, training custom classes with fewer samples only will lead to better results.
+
+Three steps to train custom classes:
+
+- Step 1: Select custom classes from original classes, named `custom_classes`. Class `0` should not be selected since it is designed for potential usage(not implemented yet) and will be added automatically.
```

Comment:
What does `potential usage` mean?

btw space before opening bracket `(`

Reply:
Actually, I saw @kennymckormick mentioned `potential usage` somewhere...

Reply:
https://github.com/open-mmlab/mmaction2/issues/508#issuecomment-753738568

I'll add the message here.